### PR TITLE
Add new sign up fields

### PR DESCRIPTION
**Pull request by stephanpavlovic**
For Marketing and Customer Support it would be great two know the company name and a contact phone number.

Bildschirmfoto 2020-10-16 um 13 53 09
Bildschirmfoto 2020-10-16 um 13 52 00

### CODE REVIEW

File: pages/business/setup/signup.tsx [hunk 1]
```diff
@@ -91,6 +95,10 @@ const SetupSignupPage: React.FC<WithOwnerProps> = () => {
             <Form>
               <Input name="name" label={isCareEnv ? 'Ihr Name' : 'Dein Name'} />
               <Box height={4} />
+              <Input name="companyName" label="Name Ihres Unternehmens" />
```
Comment (bonflintstone) [inline pages/business/setup/signup.tsx:98]:
```suggestion
              <Input name="companyName" label={isCareEnv ? 'Name Ihres Unternemen' : 'Name von deinem Unternehmen' />
```
Not sure if thats the best possible labels, but we should  keep it consistent with the formal/ informat style

**Review by bonflintstone — APPROVED** [review]
Seems like prettier still has some complaints, but apart from that it looks good to me :smile:

File: pages/business/setup/signup.tsx (hunk 1 above)
Comment (stephanpavlovic) [reply pages/business/setup/signup.tsx:98]: Hab ich lokal schon angepasst. Aber ich versuche mein eslint zu fixen :(

Comment (timomeh) [reply pages/business/setup/signup.tsx]: Should automatically be fixable with `npm run lint -- --fix`

**Pull request merged**